hack/test-go.sh

```diff
@@ -303,6 +309,9 @@ reportCoverageToCoveralls() {
   fi
 }
 
+# Start etcd
+kube::etcd::start
```

Comment:
FYI, the example above was crib'd from integration.  

ok, I'll re-jig to standup and teardown on a per-test.